applications/StructuralMechanicsApplication/custom_constitutive/beam_constitutive_law.cpp

```diff
@@ -74,27 +74,25 @@ int BeamConstitutiveLaw::Check(
     const ProcessInfo& rCurrentProcessInfo
 ) const
 {
-    if(YOUNG_MODULUS.Key() == 0 || rMaterialProperties[YOUNG_MODULUS] <= 0.00)
+    if (rMaterialProperties[YOUNG_MODULUS] <= 0.0)
```

Comment:
looking at those again, this can simply use `KRATOS_ERROR_IF`

up to you if you wanna change it